docs/source/guides/crosshairmodding.rst

```diff
@@ -0,0 +1,148 @@
+Crosshair Modding
+------------------
+
+
+Example Mod
+^^^^^^^^^^^^^^^^^^^^^
+`Custom.Crosshairs <https://github.com/MysteriousRSA/Custom.Crosshairs>`__
```

Comment:
I think that a small text here, like "If you want to know what a crosshair mod looks like, take a look at this example mod" would help way more. If you do add a text, the title can be removed as i think its a bit too much to have 3 titles visible on a single page, especially if they dont each have a proper section of text.